### DISCUSSION TITLE
posts/my-painful-but-worthwhile-migration-from-wordpress-to-jekyll/

### DISCUSSION THREAD

**Discussion by giscus** [seanthegeek/seanthegeek.github.io#1] [giscus]
# posts/my-painful-but-worthwhile-migration-from-wordpress-to-jekyll/

A journey from a bloated dynamic blog to a svelte static blog

https://seanthegeek.net/posts/my-painful-but-worthwhile-migration-from-wordpress-to-jekyll/

**Comment by seanthegeek** [comment]
Hello world!
